CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to the "vscode-java-debugger" extension will be documented i
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.7.0 - 2018-3-15
+### Added
+- Support [conditional breakpoints](https://github.com/Microsoft/vscode-java-debug/issues/118). See [PR#153](https://github.com/Microsoft/java-debug/pull/153), [PR#154](https://github.com/Microsoft/java-debug/pull/154), [PR#156](https://github.com/Microsoft/java-debug/pull/156)
+
+### Changed
+- Fix an unsupported breakpoint at method entry/exit issue. See [PR#129](https://github.com/Microsoft/java-debug/pull/129)
+- Fix the issue when projectName is not specified, expression evaluation doesn't work. See [PR#156](https://github.com/Microsoft/java-debug/pull/156)
```

Comment:
Fix the issue when `the` projectName is not specified, `the` expression evaluation doesn't work.